client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -94,21 +119,43 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 				{ translate( 'Pick an email solution' ) }
 			</h1>
 
+			<div className="email-providers-stacked-comparison__how-they-compare">
+				{ translate( 'Not sure how to start?' ) }
+				<Button
+					borderless
+					className="email-providers-stacked-comparison__how-they-compare-link"
+					onClick={ () => setInDepthComparison( ! inDepthComparison ) }
```

Comment:
Please let's redirect the user to a new page. The `Email Comparison` page and the `In Depth Comparison` page should have their own url, and should be responsible for loading any data they require. Let's have a clear separation of responsibilities.

Reply:
I'll do it for the next PR if that's fine?